techmatters/flex-plugins
# CHI-1421: Run a local HTTP server to load form definitions for local dev

## Description

* Adds an option to set the form definition base URL from an environment variable or Service Configuration
* For `npm run dev`, runs an instance of `http-server` pointing at the form definitions root folder and points the flex instance at it for form defs
* Added `dev:s3-forms` script which gives the old `dev` behaviour

### Checklist
- N/A New tests added
- N/A Feature flags added
- N/A Strings are localized

### Related Issues
Fixes #....

### Verification steps

* Run `npm run dev` and verify form defs come from a local server
* Run `npm run dev s3-forms` and verify form defs come from the remote assets URL
* Deploy flex to Aselo Development and check it still works
